### DISCUSSION TITLE
v1.0-beta.1

### DISCUSSION THREAD

**Discussion by antoine2711** [OpenRefine/SparqlExtension#28] [Announcements]
This is the first beta version of the SPARQL Importer for OpenRefine.

It should be compatible with OpenRefine, v3.5 and +.

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenRefine/SparqlExtension/releases/tag/v1.0-beta.1'>v1.0-beta.1</a>.</em>

**Comment by ostephens** [comment]
For some reason that link (which I assume was automatically generated by GitHub?) doesn't work. I think the correct link is https://github.com/OpenRefine/SparqlExtension/releases/tag/v1.0-beta.1

**Reply by antoine2711** [reply]
@ostephens: I've added the file from the mvn package. What do you think of that?

<kbd>![image](https://user-images.githubusercontent.com/12217525/183938318-5c3e5002-288a-4813-8628-87b6006f2d59.png)</kbd>

Regards, Antoine
